server/internal/adapter/gql/loader_user.go

```diff
@@ -43,7 +44,13 @@ func (c *UserLoader) SearchUser(ctx context.Context, nameOrEmail string) (*gqlmo
 		return nil, err
 	}
 
-	return gqlmodel.ToUserFromSimple(res), nil
+	for _, user := range res {
+		if user.Name == nameOrEmail || user.Email == nameOrEmail {
+			return gqlmodel.ToUserFromSimple(user), nil
+		}
+	}
+
+	return nil, errors.New("user not found.")
```

Comment:
This is not part of this PR

Reply:
Ahh, Must've been from the other changes to reearthx, Please add them back in that case.

Reply:
yes, I reverted it!